### DISCUSSION TITLE
Last version (1.56.0) extremely unstable?

### DISCUSSION THREAD

**Discussion by mauricioszabo** [atom/atom#22327] [General]
Hi, I don't know if it's only me, but is the latest Atom version (1.56.0) hugely unstable?

First, syntax highlighting is giving me trouble (it's making special distinction for things that look like URLs, and sometimes not highlighting regexp for Clojure files); then, it sometimes freeze. Simply this, no warning, nothing.

I'm not sure what's happening - older version was so stable that I left Atom open for days, and everything was fine - now, sometimes, I have to restart it every 2 hours or so...

**Comment by tiredkangaroo** [comment]
I agree! Sometimes it freezes for me at random points. Sometimes whenever I press a certain character, for example "c", it will freeze, and the way to unfreeze it is by pressing <Ctrl + O> (Open file, there is no need to actually open the file though.)

**Reply by sadick254** [reply]
@mauricioszabo @KokoTheBest we try our best to keep Atom stable as much as we can. I think it would be beneficial if you could open an issue that we can keep track of. This would help us take steps to fix the issues outlined here.

**Reply by mauricioszabo** [reply]
@sadick254 the problem is that I can't really reproduce, so I'm not sure how would I open the issue...

**Reply by mauricioszabo** [reply]
@sadick254 By the way, I'm not seeing the same problem on Atom 1.57.0-beta0

**Reply by sadick254** [reply]
@mauricioszabo Am happy that you are not experiencing the issue on Atom 1.57.0-beta0. Watch out for a new release coming soon.

**Comment by mauricioszabo** [comment]
Just to complete this discussion, Atom 1.57 is fine again :).

Had me worried :laughing:

**Reply by mauricioszabo** [reply]
Ok, I spoke too soon. It's more stable, but still freezing.

It's happening when I render a block decoration that needs to scroll too much to the side. Then, it freezes. It was not happening on Atom 1.55. Will open a bug
